src/Filter/FilterComposite.php

```diff
@@ -124,46 +124,43 @@ public function removeFilter(string $name) : void
      */
     public function filter(string $property) : bool
     {
-        $andCount = count($this->andFilter);
-        $orCount = count($this->orFilter);
-        // return true if no filters are registered
-        if ($orCount === 0 && $andCount === 0) {
+        return $this->atLeastOneOrFilterIsTrue($property) && $this->allAndFiltersAreTrue($property);
+    }
+
+    private function atLeastOneOrFilterIsTrue(string $property)
+    {
+        if (\count($this->orFilter) === 0) {
```

Comment:
we tend to import internal php functions so then it is not needed to use `\` before function name.
see top of the file: `use function count;` (line 17).

the same comment for all other internal functions usage below

Reply:
I applied it but I am curious to why to use the import rather than the `\` namespace ?

Reply:
We decided to go that way as for the performance is exactly the same, and:
- all imports are in one place
- for old libraries if we decide to add it changes will be only on top of the file, not in multiple places, so history should be clearer as well.